src/app/infrastructure/Validators.ts

```diff
@@ -1,40 +1,38 @@
 export const VALIDATOR_MESSAGES = {
-    INVALID_INTEGER: "Must be an integer.",
-    INVALID_NUMBER: "Must be a number.",
-    REQUIRED_VALUE: "This is a required field."
-    
-}
+    INVALID_INTEGER: 'Must be an integer.',
+    INVALID_NUMBER: 'Must be a number.',
+    REQUIRED_VALUE: 'This is a required field.',
+    INVALID_INTEGER_RANGE: 'Must be an integer between 1 and 50'   
+};
 
-export const required = (value?:any) => value || value===false ? undefined :  VALIDATOR_MESSAGES.REQUIRED_VALUE;
+export const required = (value?: any) => value || value === false ? undefined :  VALIDATOR_MESSAGES.REQUIRED_VALUE;
 
-
-export const integer = (value:any) =>{ 
+export const integer = (value: any) => {  
     let invalidMessage = number(value);
-    if(!invalidMessage){
+    if (!invalidMessage) {
         const numericValue = Number(value);
-        if(Math.floor(numericValue)===numericValue){
+        if (Math.floor(numericValue) === numericValue) {
             invalidMessage = undefined;
-        }else{
+        } else {
              invalidMessage = VALIDATOR_MESSAGES.INVALID_INTEGER;
         }
     }
     return invalidMessage;
-}
-
-export const number = (value:any) =>{ 
-    return value && isNaN(value) ? VALIDATOR_MESSAGES.INVALID_NUMBER : undefined;
-}
+};
 
-/* examples of other validators that can be used as needed
+export const integerBetween1and50 = (value: any) => {
```

Comment:
So you could likely compose this validator of a couple different validators: i.e.
```ts

// Notice here that we have a function that returns another function (the Validator) using the closure of the
// outer function to hold the minValue
const minValidator = (minValue:Number)=> (
    (value?:Number)=>(
        value && value < minValue ? `Must be at least ${minValue}` : undefined
    )
)

const maxValidator = (maxValue:Number)=> (
   (value?:Number) => (
       value && value > maxValue ? `Must be less than ${maxValue}` : undefined
   )
)

// could create in the validators file for common ones (see below)
const minValue1 = minValue(1)

```
>Later On...
```ts
<Field name="age" type="number"
  component={renderField} label="Age"
  validate={[minValue1,maxValidator(18) ]} // Or you should be able to use it inline as well
/>
```

An advantage to something like this is that it's more generic / general and can be re-used more easily.  Also you could create a range validator that just uses the min/max validators under the hood.  Thus composing functionality from smaller building blocks.